infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -166,7 +166,7 @@ locals {
   k => merge(v, { example_calls : v.example_api_calls }) }
 
 
-  jira_cloud_id                    = coalesce(var.jira_cloud_id, "YOUR_JIRA_CLOUD_ID")
+  jira_example_cloud_id                    = coalesce(var.jira_cloud_id, "YOUR_JIRA_CLOUD_ID")
```

Comment:
Minor refactor here for naming, as now this is only for the example calls. But if I change this across all the modules is going to be a breaking change. Another option is to duplicate the variable and choose the not null; not sure what approach you want to follow for this @eschultink